orchestration/templates/ingest-clinvar-archive.yaml

```diff
@@ -180,6 +180,34 @@ spec:
                   value: '{{ "{{tasks.ingest-raw-xml.outputs.parameters.file-id}}" }}'
                 - name: upload-path
                   value: '{{ printf "gs://%s/{{inputs.parameters.gcs-prefix}}/processed/release_history/" .Values.gcs.bucketName }}'
+          # Ingest the staged release_history row.
+          - name: ingest-release-history
+            dependencies: [stage-release-history]
+            # Same WHEN as step above.
```

Comment:
nitpick: in anticipation of things changing, instead of saying "as step above" can we reference the step's name?